docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -32,52 +32,67 @@ The following diagram provides an overview of the key GX Cloud architecture comp
+- **GX Agent** - A utility that runs in your organization's deployment environment. While running, the GX Agent can receive tasks generated from the GX Cloud web UI, such as running a Checkpoint or fetching Metrics, and execute these tasks against your Data Assets.
 
-## GX Cloud deployment patterns
+## GX Agent
+
+The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data, all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.
 
-GX Cloud deployments can be tailored to meet your specific business requirements. The following factors determine what deployment pattern is best for your organization:
+The GX Agent is [typically deployed in your organization's deployment environment](./connect/connect_lp) and serves all GX Cloud users within your organization. It can be run as part of your development or production workflows.
+
+## GX Cloud deployment patterns
 
-- Whether you want to interact with the GX OSS client locally, or use only the GX Cloud web UI.
+GX Cloud deployments can be tailored to meet your specific business requirements. To determine which deployment pattern and GX architectural components are most suitable for your organization, determine how your data is accessed and how users interact with GX Cloud.
 
-- Where and how you want to run the GX Agent. The GX Agent can be run in your local environment as a Docker container (the recommended approach), or as a Python program.
+The following table shows the connection options.
 
-### Run the GX Agent locally and interact with the GX Cloud web UI only
+| Deployment pattern | Environment | GX Agent | GX OSS client |
+| :-- | :-- | :--: | :--: |
+| Org-hosted | Deployment | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Self-hosted | Local | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Orchestrated | Deployment or local | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> |
 
-GX Cloud supports the creation of Snowflake Data Sources and Data Assets. If you're using GX Cloud to test and validate Snowflake data, a deployment pattern that allows you to run the GX Agent locally and use the GX Cloud web UI to manage your GX workflows is recommended.
+The following table shows how users can access GX Cloud.
 
-![GX deployment in which you create and manage Snowflake Data Assets only, interact with the GX Cloud web UI only, and run the GX Agent locally](./architecture_deployment_images/deployment_web_ui_only_snowflake.png)
+| Access method | GX Cloud | GX OSS client |
+| :-- | :--: | :--: |
+| UI | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Programmatic | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div>  |
```

Comment:
Do we want to say Programmatic? Would "Python" or "Python API" be more clear?

Reply:
I think "Python API" makes sense, good suggestion! I'll update this.